packages/components/src/ExampleComponent/ExampleComponent.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import { ExampleComponent } from "."
+import "@testing-library/jest-dom"
```

Comment:
```suggestion
```

Shouldn't be needed now since we should have global config to apply to all our test files